marklogic-client-api/src/test/java/com/marklogic/client/test/RowManagerTest.java

```diff
@@ -1479,6 +1485,57 @@ public void testBucketGroup() {
     JsonNode actual = row.getContainer("r");
     assertEquals("group unequal", expect, actual);
   }
+
+  @Test
+  public void testFromDocUrisWithWordQuery() {
+    RowManager rowMgr = Common.client.newRowManager();
+    PlanBuilder p = rowMgr.newPlanBuilder();
+    PlanBuilder.ExportablePlan builtPlan = p.fromDocUris(p.cts.wordQuery("trumpet"), "");
+
+    RowSet<RowRecord> recordRowSet = rowMgr.resultRows(builtPlan);
+    Set<String> uriSet = new HashSet<>();
+    uriSet.add("/optic/test/musician4.json");
+    uriSet.add("/optic/test/musician1.json");
+    Iterator<RowRecord> rows = rowMgr.resultRows(builtPlan).iterator();
+    while (rows.hasNext()){
+      String temp = rows.next().get("uri").toString().replaceAll("^\"|\"$", "");
```

Comment:
What's the replaceAll needed for?

Reply:
replaceAll will get rid of the double quotes and the variable temp will be - /optic/test/musician4.json instead of "/optic/test/musician4.json". Without the replaceAll the asserts will fail.